precompiles/xvm/src/tests.rs

```diff
@@ -23,6 +23,7 @@ use astar_primitives::xvm::CallError;
 use parity_scale_codec::Encode;
 use precompile_utils::testing::*;
 use precompile_utils::EvmDataWriter;
+use sp_core::U256;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `U256` type from `sp_core` is being used in this section. Ensure that the necessary import for `U256` from `sp_core` is available. If not, add the import to prevent compilation errors.

```diff
use sp_core::U256;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->